src/backend/joanie/core/models/products.py

```diff
@@ -806,10 +868,24 @@ def clean(self):
                     )
                 else:
                     nb_seats = self.order_group.nb_seats
-                    if 0 < nb_seats <= self.order_group.get_nb_binding_orders():
-                        error_dict["order_group"].append(
-                            f"Maximum number of orders reached for product {product_title:s}"
-                        )
+                    if nb_seats is not None:
+                        if 0 <= nb_seats <= self.order_group.get_nb_binding_orders():
```

Comment:
Could be simplified to
```suggestion
                        if nb_seats == self.order_group.get_nb_binding_orders():
```